interfaces/builtin/opengl_test.go

```diff
@@ -135,3 +149,20 @@ func (s *OpenglInterfaceSuite) TestAutoConnect(c *C) {
 func (s *OpenglInterfaceSuite) TestInterfaces(c *C) {
 	c.Check(builtin.Interfaces(), testutil.DeepContains, s.iface)
 }
+
+func (s *OpenglInterfaceSuite) TestMountSpec(c *C) {
+	tmpdir := c.MkDir()
+	dirs.SetRootDir(tmpdir)
```

Comment:
If you're mocking the `dirs` variables, you should clean up after the test. Copying `DesktopInterfaceSuite.TearDownTest` to this suite would be an easy way of doing this.